api/v1beta1/foundationdb_version.go

```diff
@@ -208,8 +202,7 @@ var Versions = struct {
 	WithBinariesFromMainContainer:        FdbVersion{Major: 6, Minor: 2, Patch: 15},
 	WithoutBinariesFromMainContainer:     FdbVersion{Major: 6, Minor: 2, Patch: 11},
 	WithRatekeeperRole:                   FdbVersion{Major: 6, Minor: 2, Patch: 15},
-	WithoutRatekeeperRole:                FdbVersion{Major: 6, Minor: 1, Patch: 12},
 	WithSidecarCrashOnEmpty:              FdbVersion{Major: 6, Minor: 2, Patch: 20},
 	WithoutSidecarCrashOnEmpty:           FdbVersion{Major: 6, Minor: 2, Patch: 15},
-	MinimumVersion:                       FdbVersion{Major: 6, Minor: 1, Patch: 12},
+	MinimumVersion:                       FdbVersion{Major: 6, Minor: 2, Patch: 0},
```

Comment:
I think a good minimum version would be `6.2.20` that would allow us to remove nearly all of the version specific behaviour.

Reply:
The minimum version should be 6.3.22, try to find the documentation in the forum and document it.